experimental/framework/tinylicious-client/src/test/TinyliciousClient.spec.ts

```diff
@@ -32,15 +149,193 @@ describe("TinyliciousClient", () => {
                 map1: SharedMap,
             },
         };
+        const [container] = await TinyliciousClient.createContainer(containerConfig, schema);
+        await new Promise<void>((resolve, reject) => {
+            container.on("connected", () => {
+                resolve();
+            });
+        });
+        assert.notStrictEqual(container, undefined, "Container cannot be created");
+    });
 
+    /**
+    * Scenario: Given the container already exists, test that TinyliciousClient can get the existing container
+    * when provided with valid ContainerConfig and ContainerSchema.
+    *
+    * Expected behavior: containerCreate should have the identical SharedMap ID as containerGet.
+    */
+    it("can get a container successfully", async () => {
+        const containerConfig: TinyliciousContainerConfig = { id: documentId };
+        const schema: ContainerSchema = {
+            name: documentId,
+            initialObjects: {
+                map1: SharedMap,
+            },
+        };
+        const [containerCreate] = await TinyliciousClient.createContainer(containerConfig, schema);
+        await new Promise<void>((resolve, reject) => {
+            containerCreate.on("connected", () => {
+                resolve();
+            });
+        });
+        const [containerGet] = await TinyliciousClient.getContainer(containerConfig, schema);
+        const map1Create = containerCreate.initialObjects.map1 as SharedMap;
+        const map1Get = containerGet.initialObjects.map1 as SharedMap;
+        assert.strictEqual(map1Get.id, map1Create.id, "Error getting a container");
+    });
+
+    /**
+    * Scenario: test if given the appropriate COntainerConfig and ContainerSchema,
+    * TinyliciousClient.createContainer will return the a valid ContainerServices.
+    *
+    * Expected behavior: the returned containerServices should not be undefined
+    */
+    it("can get containerServices successfully", async () => {
+        const containerConfig: TinyliciousContainerConfig = { id: documentId };
+        const schema: ContainerSchema = {
+            name: documentId,
+            initialObjects: {
+                map1: SharedMap,
+            },
+        };
+        const [,containerService] = await TinyliciousClient.createContainer(containerConfig, schema);
+        assert.notStrictEqual(containerService, undefined, "ContainerServices invalid");
```

Comment:
same feedback as above about promises and undefined

Reply:
if we are testing to see if an error is thrown or a promise is rejected, how do we differentiate between checking valid container and valid services?

Reply:
if you mean checking one independently of the other, you can't with this api because the two are paired in the same promise.  the typing means we should expect that if there's a problem with one, the entire promise rejects even if the other is fine

Reply:
I see. In this case, checking for valid containerServices would be redundant, so I'll remove the test case.